src/plugins/views/PluginList/filters.ts

```diff
@@ -35,10 +35,10 @@ export function getFilterVariables(
   params: PluginListUrlFilters
 ): PluginFilterInput {
   return {
-    active:
-      params.active !== undefined
-        ? parseBoolean(params.active, true)
-        : undefined,
+    // active:
+    //   params.active !== undefined
+    //     ? parseBoolean(params.active, true)
+    //     : undefined,
```

Comment:
🚓

Reply:
This part of code overlaps with [plugin details pr](https://github.com/mirumee/saleor-dashboard/pull/1083) and types would fail miserably if it wasn't commented out. Will be removed in feature branch